src/app/components/ContactPlaceholder.js

```diff
@@ -1,18 +1,24 @@
 import React from 'react';
-import { Link } from 'react-router-dom';
 import PropTypes from 'prop-types';
 import { c, msgid } from 'ttag';
-import { PrimaryButton, Button, Icon, useModals, useContactGroups } from 'react-components';
+import { PrimaryButton, Button, Icon, Href, useModals, useContactGroups } from 'react-components';
+import { withRouter } from 'react-router-dom';
```

Comment:
`ContactPlaceholder` is used in `ContactsContainer` where you have `location, history`, so you can pass it as a prop instead of using `withRouter` here.